providers-config.default.toml

```diff
@@ -20,3 +20,9 @@
 	pre_cmd = "mkdir -p $volume/backups && slapcat > $volume/backups/all.ldif"
 	detect_cmd = "[[ -f $volume/DB_CONFIG ]]"
 	backup_dir = "backups"
+	
+	[providers.mongo]
+	pre_cmd = "mongodump -o $volume/backups"
```

Comment:
Shouldn't you create the `backups` directory first, like what is done on other providers?

Reply:
The ‘mongodump’ command creates the backup directory.

Reply:
OK. Then it's OK for me to merge this. @cryptobioz @raphink any objection?

Reply:
LGTM